api-inference-community/docker_images/spacy/requirements.txt

```diff
@@ -0,0 +1,3 @@
+starlette==0.14.2
+api-inference-community==0.0.9
```

Comment:
This doesn't exist, does it ?

How should we go about it ?

Reply:
Oops, downgraded to 0.0.6 which exists